tests/end2end_tests/entrypoints/sshnpd_entrypoint.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 echo "SSHNPD START ENTRY"
-SSHNPD_COMMAND="$HOME/.local/bin/sshnpd -a @sshnpdatsign -m @sshnpatsign -d deviceName -s -u -v 2> err.txt"
+SSHNPD_COMMAND="$HOME/.local/bin/sshnpd -a @sshnpdatsign -m @sshnpatsign -d deviceName -s -u -v 2>&1 | tee all.txt"
 echo "Running: $SSHNPD_COMMAND"
 eval "$SSHNPD_COMMAND"
+tail -f all.txt
```

Comment:
Doesn't doing `tee all.txt` and `tail -f all.txt` duplicate the logs?
tee writes to files and stdout, which means you don't need to tail the file as well...


```
~ 
➜ echo hello | tee world
hello

~ 
➜ tail -f world                        
hello
```
In this example, hello is echoed once, printed twice...

Reply:
You should either do `<command> > all.txt 2>&1; tail -f all.txt` or `<command> 2>&1 all.txt | tee all.txt`... unless something weird is happening in the runner?

Reply:
The `tail -f` in the entrypoint never actually worked at all, and I forgot to remove it. 

Just removed it in this commit https://github.com/atsign-foundation/sshnoports/pull/449/commits/1bfd62c5ba7c2cbcb7696a0d6a0138c26af954a1

Reply:
Ah probably because we are in a pty